src/core/fbs/mutator/fetcher.ts

```diff
@@ -52,7 +52,7 @@ export const fetcher = async <ResponseType>({
   data?: BodyType<unknown>;
   signal?: AbortSignal;
 }) => {
-  const libraryToken = getToken(TOKEN_LIBRARY_KEY);
+  const libraryToken = getToken(TOKEN_USER_KEY);
```

Comment:
This cannot be added in the current form. This change will mean that requests against FBS for e.g. availability information will fail because of missing tokens without a user being logged in.

One way to address this would be to add the user token if that is available and fall back to the library token which should always be available.

Also the variable name `libraryToken` is misleading.

Reply:
Fixed